frontend/src/app/testQueue/AoEForm/PriorTestInputs.tsx

```diff
@@ -161,8 +170,10 @@ const PriorTestInputs: React.FC<Props> = ({
               // Fill in last test info using this data
               // TODO: update when test history has test type
               setPriorTestType("2");
-              setPriorTestDate((lastTest.dateTested || "").split("T")[0]);
-              setPriorTestResult(lastTest?.result);
+              setPriorTestDate(
+                (lastTest.dateTested || "").split("T")[0] as FormattedDate
+              );
+              setPriorTestResult(lastTest?.result as TestResult);
```

Comment:
It seems that `lastTest.result` is the incorrect type. Can you update it from a `string` to `TestResult` so we don't have to do this casting here

Reply:
Done in 7026199.